sripwoud/rideau
# feat(client): show list of joined groups

## Test plan
1. login with a pse.dev address
   1. enter mail address in input field
   2. click link in received email
   3. page should show a loader and after "logged as <email>"
   4. click on dashboard link
   5. in the left column you should see that you are part of the "@pse.dev rideau test" group
2.  Login with non pse.dev address
  Repeat previous steps. Login should be successful. You can see the dashboard. But you are not part of any groups. You should not see the "@pse.dev rideau test" group on the left side
